### ISSUE TITLE
Emergency shutter 2x1 and 3x1 need proper replacing with these new sprites.

### ISSUE THREAD

**Issue by JakeDaBoss** [Lykayos/Clean-Bay-copy-No-Commit-History#98]
![shutter_open_3x1_final](https://cloud.githubusercontent.com/assets/2639996/13899042/6e06d45c-ede4-11e5-9b93-6838032973fb.gif)
![shutter_welded_2x1_final](https://cloud.githubusercontent.com/assets/2639996/13899044/6e075472-ede4-11e5-9699-8705ad82fbc4.png)
![shutter_welded_3x1_final](https://cloud.githubusercontent.com/assets/2639996/13899043/6e072650-ede4-11e5-9c95-dcf7fc335ecf.png)
![shutter_alert_2x1_final](https://cloud.githubusercontent.com/assets/2639996/13899047/6e08e10c-ede4-11e5-8102-bf06e1d2c76c.gif)
![shutter_alert_3x1_final](https://cloud.githubusercontent.com/assets/2639996/13899045/6e07e5c2-ede4-11e5-97b3-1d0619e56f13.gif)
![shutter_open_2x1_final](https://cloud.githubusercontent.com/assets/2639996/13899046/6e07eee6-ede4-11e5-8dfa-0f3fc2cba3d0.gif)

INSTRUCTIONS:

The gifs contain the opening and closing animations INSIDE. You need to convert these to .DMI (just change the extension), and then copy / paste them one by one into the actual animation in the official 3x1 and 2x1 .dmi file. Keep in mind these only have 6 frames or less (unsure), so you'll have to lower them from the original 8.

Thanks to anyone doing this.

**Comment by Nightspark43** [comment]
Imma work on this.

**Comment by Nightspark43** [comment]
Yeah I'm having issues getting this, apparently even PORTING icons is out of my scope, somehow.
